src/installation.rst

```diff
@@ -30,47 +30,69 @@ present Cylc only tested on Linux.
 Quick Installation
 ------------------
 
-Via Conda (recommended):
+Conda (recommended)
+^^^^^^^^^^^^^^^^^^^
+
+Base installation (just the :term:`scheduler`):
 
 .. code-block:: console
 
-   # install cylc with the browser-GUI
    $ conda install cylc
 
-Via pip:
+Full installation with the UI Server and Rose
+:ref:`Cylc components <Cylc components>`.
 
-.. note::
+.. code-block:: console
 
-   We recommend using a virtual environment.
+   $ conda install cylc cylc.uiserver cylc.rose
 
-.. note::
+Pip
+^^^
+
+.. warning::
+
+   You must ensure :ref:`non-Python dependencies` have been installed via
+   other means.
```

Comment:
Mention that a built cylc-ui is now included in cylc-uiserver?

Reply:
(OK I see you're not including cylc-ui in the list of non-Python deps, further down).